src/drop.js

```diff
@@ -29,5 +26,5 @@ var slice = require('./slice');
  *      R.drop(3, 'ramda');               //=> 'da'
  */
 module.exports = _curry2(_dispatchable('drop', _xdrop, function drop(n, xs) {
-  return slice(Math.max(0, n), Infinity, xs);
+  return _slice(xs, Math.max(0, n));
```

Comment:
Seeing negative indexes aren't supported on `slice` the `Math.max` isn't necessary